pr_testing/test_multiple_prs.sh

```diff
@@ -467,7 +467,7 @@ voms-proxy-init -voms cms -valid 24:00 || true  # To get access to jenkins artif
 
 DO_COMPARISON=true
 DO_MB_COMPARISON=false
-if [ "$TEST_CONTEXT" = "GPU" ] ; then
+if [ "$TEST_CONTEXT" = "GPU" -o "$TEST_CONTEXT" = "PROFILING" ] ; then
```

Comment:
this change is not needed. It is only for GPUs for which which run two tests, one normal and one only for GPUs. For profiling we are ging to run normal tests with additional profiling test